dlrs/main/classes/RollupSummariesSelector.cls

```diff
@@ -167,99 +219,6 @@ public class RollupSummariesSelector {
       return LookupRollupSummary__c.sObjectType;
     }
 
-    public List<RollupSummary> selectById(Set<String> idSet) {
```

Comment:
Now that we're not running queries for specific use cases we are able to remove nearly all of these per-type queries.